core/engine/src/builtins/temporal/now.rs

```diff
@@ -83,25 +85,40 @@ impl Now {
             .map(|v| to_temporal_timezone_identifier(v, context))
             .transpose()?;
         create_temporal_datetime(
-            NowInner::plain_date_time_with_provider(tz, context.tz_provider())?,
+            NowInner::plain_datetime_iso_with_provider(tz, context.tz_provider())?,
             None,
             context,
         )
         .map(Into::into)
     }
 
     /// `Temporal.Now.zonedDateTime`
-    fn zoned_date_time(_: &JsValue, _: &[JsValue], _: &mut Context) -> JsResult<JsValue> {
-        Err(JsNativeError::error()
-            .with_message("not yet implemented.")
-            .into())
+    fn zoneddatetime(_: &JsValue, args: &[JsValue], context: &mut Context) -> JsResult<JsValue> {
```

Comment:
Thought: not gonna bikeshed because I think it's not worth for internal APIs, but we should probably do a pass through our names to agree on at least a "convention" .

Reply:
Yeah, I've been wanting to open an issue to go over naming conventions in `temporal_rs` for a few weeks now. Would be handy for Boa too